storage/mysql/storage.sql

```diff
@@ -4,23 +4,40 @@
 -- Tree stuff here
 -- ---------------------------------------------
 
+-- Enable strict mode, so invalid data on inserts/updates is treated as error
+-- instead of warning.
+-- https://dev.mysql.com/doc/refman/5.7/en/sql-mode.html#sql-mode-strict
+SET GLOBAL sql_mode = 'STRICT_ALL_TABLES';
 
 -- Tree parameters should not be changed after creation. Doing so can
 -- render the data in the tree unusable or inconsistent.
+-- TODO(codingllama): Add NOT NULL constraints to most columns
 CREATE TABLE IF NOT EXISTS Trees(
-  TreeId                INTEGER NOT NULL,
+  TreeId                BIGINT NOT NULL,
```

Comment:
We picked int64 for tree IDs, which don't fit on a MySQL integer.

Fun fact: non-strict MySQL silently truncates int64 to int32, which caused a bunch of my IDs to become int32_max_value. Good times.